src/temporal/t.rast.climatologies/t.rast.climatologies.html

```diff
@@ -0,0 +1,32 @@
+<h2>DESCRIPTION</h2>
+<b>t.rast.climatologies</b> is a module to calculate climatologies, i.e., long term stats, for single days or months
+in a time series. If the <em>s</em> flag is used, the module outputs a new space time raster dataset
+of relative temporal type with the aggreated maps.
+
+<h2>EXAMPLE</h2>
+
+<h3>Daily climatologies</h3>
+Starting from a space time raster dataset of daily granularity (or granularity lower than one day), the module will create a new space time raster dataset of relative temporal type containing the long term average for each day along years.
+<div class="code">
+    <pre>
+        t.rast.climatologies input=myinput output=dailyoutput
+    </pre>
+</div>
+
+<h3>Monthly climatologies</h3>
+Starting from a space time raster dataset of monthly granularity (or lower than one month), the module will create two new space time raster datasets containing the long term minimum and maximum for each month along years.
```

Comment:
```suggestion
<h2>DESCRIPTION</h2>

<b>t.rast.climatologies</b> is a module to calculate climatologies, i.e., long term stats, for single days
or months in a time series. If the <em>s</em> flag is used, the module outputs a new space time
raster dataset of relative temporal type with the aggregated maps.

<h2>EXAMPLE</h2>

<h3>Daily climatologies</h3>

Starting from a space time raster dataset of daily granularity (or granularity lower than one day),
the module will create a new space time raster dataset of relative temporal type containing the
long term average for each day along years.

<div class="code">
    <pre>
        t.rast.climatologies input=myinput output=dailyoutput
    </pre>
</div>

<h3>Monthly climatologies</h3>

Starting from a space time raster dataset of monthly granularity (or lower than one month),
the module will create two new space time raster datasets containing the long term minimum
and maximum for each month along years.
```

(breaking long lines)